### ISSUE TITLE
Type definition of type '*ast.InterfaceType' is not supported yet. Using 'object' instead.

### ISSUE THREAD

**Issue by hyprnick** [swaggo/swag#850]
```go
type Tag struct {
	ID    int64       `json:"id"`
	Name  string      `json:"name"`
	Value interface{} `json:"value"`
}
```

When I run `swag init`, I get the following warning: 

`Type definition of type '*ast.InterfaceType' is not supported yet. Using 'object' instead.`

I would like to specify multiple options for types here as this could be a string or a float. In the future, boolean could be possible too.  

Is this possible? Thank you!  

```
swag 1.6.9
go 1.14.2
OS: Mac
```

**Comment by sdghchj** [comment]
If used in response, refer to [https://github.com/swaggo/swag#model-composition-in-response](https://github.com/swaggo/swag#model-composition-in-response).

**Comment by tangheng1995** [comment]
same questions

swag: 1.7.0
go: 1.15.6
os: debain 10

**Comment by lovechuck** [comment]
same questions 
get  info Type definition of type '*ast.InterfaceType' is not supported yet. Using 'object' instead.
but josn seems to be ok

**Comment by hapi666** [comment]
I have the same problem.
Type definition of type '*ast.InterfaceType' is not supported yet. Using 'object' instead.

**Comment by tangheng1995** [comment]
go get -u github.com/swaggo/swag/cmd/swag@v1.6.7
is ok

**Comment by PatrickChagastavares** [comment]
> go get -u github.com/swaggo/swag/cmd/swag@v1.6.7
> is ok

Congratulation!!!

**Comment by patrickalexchan** [comment]
> go get -u github.com/swaggo/swag/cmd/swag@v1.6.7
> is ok

It solved the warning issue, thankyou !

**Comment by Jeyrce** [comment]
The same question in  1.7.0 version

**Comment by fhquthpdw** [comment]
> go get -u github.com/swaggo/swag/cmd/swag@v1.6.7
> is ok

solved the issue

**Comment by ubogdan** [comment]
Either you use the latest version from the master branch, or you can wait until 1.7.2 is released.

**Comment by leason00** [comment]
> Either you use the latest version from the master branch, or you can wait until 1.7.2 is released.

when?

**Comment by ubogdan** [comment]
We don't have an eta for the release.

**Comment by ubogdan** [comment]
Good news. We have released 1.7.2 just today.
